doc/docker.md

```diff
@@ -1,16 +1,102 @@
 # Running with Docker
 
+The advantage of using docker-compose is that it is configured to run several services that the Control Panel relies on, such as the PostgreSQL database and Redis. It initializes these ready for local development purposes.
+
+## Build
+
 To build the docker image, use the following command:
+
 ```sh
 docker build -t controlpanel .
 ```
 
-Running the app requires a PostgreSQL database, and this is provided with
-docker-compose:
+## .env file
+
+You need to create a `.env` file with the settings to enable it to connect to external services.
```

Comment:
Where do I get these settings from, and what is the syntax for a `.env` file in this context..?

Reply:
I've reworded this to make more sense